browser/resources/settings/sources.gni

```diff
@@ -118,9 +118,33 @@ brave_settings_non_web_component_files = [
   "brave_web3_domains_page/brave_web3_domains_browser_proxy.ts",
   "default_brave_shields_page/brave_adblock_browser_proxy.ts",
   "default_brave_shields_page/default_brave_shields_browser_proxy.ts",
+  "getting_started_page/brave_account_browser_proxy.ts",
```

Comment:
@petemill seems like we should be using `ts_deps` just like we're using `deps` in cpp sources?

Reply:
these need to be inputs to the chromium target so that one bundle is produced without duplication (and double-module running).  But @szilardszaloki some of these look like web_component files and can be in the other list without repeating the .html.ts entry?

Reply:
I think this could only work if these were Polymer custom elements? For Lit elements, the recommended way to pass both `.ts` and `.html.ts` files is via `non_web_component_files`, as there are no corresponding `.html` files, and the `.html.ts` files are checked in.

Reply:
@petemill, as @szilardszaloki says Lit doesn't use the `web_component_files` array :smile: